### PR TITLE
Add a new tab named Info to include server endpoints details in the application view

### PR DESCRIPTION
**Pull request by Isanka95**
### Purpose
Added a new tab named **Info** to include the server endpoints details
Removed help panel in the application view as its content is now contained in the **Info** tab.

Screenshot from 2021-05-29 22-27-04

### Related Issues
- Issue [wso2-enterprise/asgardeo-produc#3582](https://github.com/wso2-enterprise/asgardeo-product/issues/3582)
- Issue [wso2-enterprise/asgardeo-produc#2743](https://github.com/wso2-enterprise/asgardeo-product/issues/2743)
- Issue [wso2-enterprise/asgardeo-produc#2080](https://github.com/wso2-enterprise/asgardeo-product/issues/2080)

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation
- [ ] Documentation provided. (Add links)
- [ ] Unit tests provided. (Add links if any)
- [ ] Integration tests provided. (Add links if any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 1]
```diff
@@ -35,14 +35,14 @@ import {
...
+import { Info } from "./settings/info";
 import { ComponentExtensionPlaceholder, applicationConfig } from "../../../extensions";
 import { AppState, CORSOriginsListInterface, FeatureConfigInterface, getCORSOrigins } from "../../core";
 import { getInboundProtocolConfig } from "../api";
 import { ApplicationManagementConstants } from "../constants";
 import {
-    ApplicationInterface,
-    ApplicationTemplateInterface,
-    AuthProtocolMetaListItemInterface,
+    ApplicationInterface, ApplicationTemplateInterface,
```
Comment (brionmario) [inline apps/console/src/features/applications/components/edit-application.tsx:44]:
```suggestion
    ApplicationInterface,
    ApplicationTemplateInterface,
```

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 2]
```diff
@@ -35,14 +35,14 @@ import {
...
+    AuthProtocolMetaListItemInterface, OIDCApplicationConfigurationInterface,
```
Comment (brionmario) [inline apps/console/src/features/applications/components/edit-application.tsx:45]:
```suggestion
    AuthProtocolMetaListItemInterface,
    OIDCApplicationConfigurationInterface,
```

File: apps/console/src/features/applications/components/settings/info.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+/**
+ * Proptypes for the server endpoints details component.
+ */
+interface InfoPropsInterface extends LoadableComponentInterface, TestableComponentInterface {
+    inboundProtocols: InboundProtocolListItemInterface[];
+    isSAMLConfigLoading: boolean;
+    isOIDCConfigLoading: boolean;
+}
+export const Info: FunctionComponent<InfoPropsInterface> = (
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/info.tsx:42]: Lets add a component doc comment here.
```suggestion

export const Info: FunctionComponent<InfoPropsInterface> = (
```

File: apps/console/src/features/applications/components/settings/info.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                    setIsSAML(true);
+                    setIsLoading(isSAMLConfigLoading);
+                }
+            });
+        }
+    }, [inboundProtocols]);
+
+    return (
+        !isLoading ? (
+            <EmphasizedSegment loading={ isLoading } padded="very">
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/info.tsx:76]: Add test id here.

```suggestion
            <EmphasizedSegment loading={ isLoading } padded="very">
```

File: apps/console/src/features/applications/components/settings/info.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+                                            "subTitle") }
+
+                            <Divider hidden/>
+                            { isOIDC && (
+                                <OIDCConfigurations oidcConfigurations={ oidcConfigurations }/>
+                            ) }
+                            { isOIDC && isSAML ? (
+                                <>
+                                    <Divider hidden/>
+                                    <Divider/>
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/info.tsx:97]: Try adding a className of `x2` or `x3`. And remove the two hidden dividers.
```suggestion
                                    <Divider className="x2"/>
```

File: apps/console/src/features/applications/components/settings/info.tsx (hunk 3 above)
Comment (Isanka95) [reply apps/console/src/features/applications/components/settings/info.tsx:59]: Added

File: apps/console/src/features/applications/components/edit-application.tsx [hunk 6]
```diff
@@ -639,10 +695,24 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
...
+                },
+                render: InfoTabPane
             }
         ];
     };
 
+    /**
+     * Handles the tab change.
+     */
+    const handleTabChange = (e, { activeIndex }) => {
```
Comment (brionmario) [inline apps/console/src/features/applications/components/edit-application.tsx:712]:
```suggestion
    const handleTabChange = (e, { activeIndex }): void => {
```

File: apps/console/src/features/applications/components/settings/info.tsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+    props: InfoPropsInterface
+): ReactElement => {
+
+    const {
+        inboundProtocols,
+        isOIDCConfigLoading,
+        isSAMLConfigLoading,
+        [ "data-testid" ]: testId
+    } = props;
+    const oidcConfigurations: OIDCApplicationConfigurationInterface = useSelector(
```
Comment (brionmario) [inline apps/console/src/features/applications/components/settings/info.tsx:69]:
```suggestion
    
    const oidcConfigurations: OIDCApplicationConfigurationInterface = useSelector(
```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/info.tsx [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    return (
+        !isLoading ? (
+            <EmphasizedSegment loading={ isLoading } padded="very" data-testid={ testId }>
+                <Grid className="form-container with-max-width">
+                    <Grid.Row>
+                        <Grid.Column>
+                            <Heading ellipsis as="h4">
+                                <strong>
+                                    { t("console:develop.features.applications.helpPanel.tabs.start.content." +
+                                        "endpoints." + "title") }
```
Comment (janakamarasena) [inline apps/console/src/features/applications/components/settings/info.tsx:106]: Unnecessary concatenation

Comment (IsankaSR) [reply apps/console/src/features/applications/components/settings/info.tsx:106]: Already fixed with PR https://github.com/wso2/identity-apps/pull/2090

**Pull request merged**